src/Divi/Modules/DonationReceipt/index.js

```diff
@@ -63,7 +63,9 @@ export default class DonationReceipt extends React.Component {
 			error: this.props.error,
 		};
 
-		API.post( `/render-donation-receipt?donation_id=${ this.props.donation_id }`, params, { cancelToken: CancelToken.token } )
+		const delimiter = ( this.props.pretty_urls === 'on' ) ? '?' : '&';
+
+		API.post( `/render-donation-receipt${ delimiter }donation_id=${ this.props.donation_id }`, params, { cancelToken: CancelToken.token } )
```

Comment:
Since this is a POST request, should we send the Donation ID as POST data, instead of a query parameter?

This would also remove the need to considering permalink settings.

Reply:
I thought the same, but it won't work. That parameter has to be in the URL because there is no other way to set it. Without it, the receipt won't even load.
https://github.com/impress-org/givewp/blob/9cf42048f970424f7ca4e208832cc7bcd7f3448d/includes/misc-functions.php#L2268

Reply:
Ah, ok. I get it now. Can you add a comment explaining the need for the work around? That way we don't break it later thinking we are doing something good.

Reply:
Resolved 
b661ba9